crates/sui-indexer-alt/src/handlers/coin_balance_buckets.rs

```diff
@@ -316,7 +322,7 @@ mod tests {
 
     // Get all balance buckets from the database, sorted by object_id and cp_sequence_number.
     async fn get_all_balance_buckets(
-        conn: &mut db::Connection<'_>,
+        conn: &mut <Db as Store>::Connection<'_>,
```

Comment:
This one is unnecessary, I think.

```suggestion
        conn: &mut Db::Connection<'_>,
```